specs/Workers.md

```diff
@@ -0,0 +1,1389 @@
+Workers support in WebView2
+===
+
+# Background
+Currently, WebView2 lacks comprehensive APIs for developers to fully utilize
```

Comment:
May want to say the following to make it clear this is adding WebView2 APIs to interact with workers and not the ability to use workers at all. That is, WebView2 currently allows use of workers but with no way to interact with them via WebView2 APIs.

```suggestion
Currently, although workers can be used in web content in WebView2, WebView2 lacks comprehensive WebView2 APIs for developers to fully utilize
```